src/servers/CMakeLists.txt

```diff
@@ -164,6 +181,17 @@ add_executable(
   ${TRACING_OBJECTS}
 )
 set_property(TARGET main PROPERTY OUTPUT_NAME trtserver)
+
+if(${TRTIS_ENABLE_GPU})
+  target_include_directories(main PRIVATE ${CUDA_INCLUDE_DIRS})
```

Comment:
I don't think you need this

Reply:
Needed since we include the cuda_runtime_api in the trtserver. (If we don't, cudaIpcMemHandle_t is not defined.